lib/Dialect/Pipeline/Transforms/ScheduleLinearPipeline.cpp

```diff
@@ -198,10 +198,33 @@ ScheduleLinearPipelinePass::schedulePipeline(UnscheduledPipelineOp pipeline) {
 
   for (auto [startTime, ops] : stageMap) {
     Block *stage = schedPipeline.getStage(startTime);
+
+    // Caching of SourceOp passthrough values defined in this stage.
+    mlir::DenseMap<Value, Value> sourceOps;
+    auto getOrCreateSourceOp = [&](OpOperand &opOperand) {
```

Comment:
nit: please be explicit about the return type of this lambda to make it easier to read. (ie add `-> Value`)